apps/wallet/src/ui/app/staking/validators/ValidatorLogo.tsx

```diff
@@ -41,24 +41,31 @@ export function ValidatorLogo({
         TRUNCATE_PREFIX_LENGTH
     );
 
-    const validatorName = useMemo(() => {
+    const validatorMeta = useMemo(() => {
         if (!validatorsData) return null;
 
         const validator = validatorsData.find(
             ({ sui_address }) => sui_address === validatorAddress
         );
         if (!validator) return null;
 
-        return getName(validator.name);
-    }, [validatorAddress, validatorsData]);
+        const logo =
+            (validator.image_url && typeof validator.image_url === 'string') ||
+            Array.isArray(validator.image_url)
+                ? validator.image_url
+                : null;
 
-    const logo = null;
+        return {
+            name: getName(validator.name),
+            logo: logo && getName(logo),
```

Comment:
Don't `getName` this, just only support string image URLs (which is the case for testnet)

Reply:
no `sui_getValidators` is array numbers

Reply:
Ugh okay but we need to clean this up after this merges.